client-linuxapp/src/main.rs

```diff
@@ -60,32 +67,25 @@ fn get_to2_urls(entries: &[TO2AddressEntry]) -> Vec<String> {
     urls
 }
 
-async fn get_client_list(
-    rv_info: Vec<RendezvousInterpretedDirective>,
-) -> Result<Vec<ServiceClient>> {
-    if rv_info.is_empty() {
-        bail!("No rendezvous entries found we can construct a client for");
-    }
+async fn get_client_list(rv_entry: &RendezvousInterpretedDirective) -> Result<Vec<ServiceClient>> {
     let mut service_client_list = Vec::new();
-    for rv_entry in rv_info {
-        let urls = rv_entry.get_urls();
-        if urls.is_empty() {
-            continue;
-        }
-        if rv_entry.bypass {
-            todo!();
-        }
-        if rv_entry.wifi_ssid.is_some() {
-            todo!();
-        }
-        if rv_entry.user_input {
-            todo!();
-        }
-        for url in &urls {
-            service_client_list.push(fdo_http_wrapper::client::ServiceClient::new(url));
-        }
-    }
 
+    let urls = rv_entry.get_urls();
+    if urls.is_empty() {
+        bail!("No URLs found in this RV_entry");
```

Comment:
This means that RV Entries that only have wifi info or request user input are not processed.

Reply:
Tru dat. instead of bailing, maybe `continue`?